package.json

```diff
@@ -11,10 +11,10 @@
   "types": "./dist/index.d.ts",
   "scripts": {
     "clean": "rimraf dist",
-    "test": "npm run lint && npm run test:karma",
-    "test:karma": "karma start --single-run --browsers ChromeHeadless karma.conf.js",
-    "lint": "eslint src test",
-    "build": "tsc src/index.d.ts && rollup -c"
+    "pretest": "tsc --project tsconfig.test.json",
+    "test": "npm run lint && karma start --single-run --browsers ChromeHeadless karma.conf.js",
```

Comment:
tests use an almost identical tsconfig that doesn't exclude tests. This is the best way I could figure out how to do this 🤷